docker-compose.dev.yml

```diff
@@ -52,6 +52,10 @@ services:
     depends_on:
       - postgrest
 
+  admin-proxy:
+    build:
+      dockerfile: ./admin-proxy/Dockerfile
```

Comment:
I'm wondering if it's better to use a docker image to install dependencies on you file system and mount the projects and the dependencies in a row node image. Doing so will remove the install step and make the app starts quicker in dev.

Reply:
I usually do not install the dependencies on the project without mounting it to avoid bugs with native dependencies (wich can happen when developper uses MacOS).